clustered/integration-test/src/test/java/org/ehcache/clustered/BasicCacheOpsMultiThreadedTest.java

```diff
@@ -66,7 +67,7 @@
 
   @ClassRule
   public static Cluster CLUSTER =
-    newCluster().in(clusterPath()).withServiceFragment(RESOURCE_CONFIG).build();
+    newCluster().in(clusterPath()).withServiceFragment(RESOURCE_CONFIG).startupBuilder(DynamicConfigStartupBuilder::new).build();
```

Comment:
@saurabhagas : we would need a way to not have to go over all the Galvan tests in the project.
ideas:
- use a META-INF/services ?
- change teh default behavior of galvan ?
- have this class moved somewhere else (in galvan) and used by the rule ?
I will let you see that with @chrisdennis

Reply:
platform needs a galvan-support module that supports it's kit.  When this is the direction we go universally, galvan-support should be removed from tc-core.

Reply:
See my comment on the class duplication below.

Reply:
@saurabhagas : I have opened TDB-4939. I will let you see with @chrisdennis  / @myronkscott about how to solve that. To me, PR works, is usable, can be merged and should be the priority.